modules/ocf_puppet/manifests/puppetserver.pp

```diff
@@ -35,6 +35,17 @@
     notify               => Service['puppetserver'],
   }
 
+  puppet_authorization::rule { 'allow-puppetserver-cli':
+    match_request_path   => '/puppet-ca/v1/(?:certificate|certificate_status)',
+    match_request_type   => 'regex',
+    match_request_method => ['get', 'post', 'put', 'delete'],
+    allow                => $::fqdn,
+    sort_order           => 998,
```

Comment:
Why does this need to be specified? The default is `200`, does this need to be after the default?

Reply:
otherwise, earlier default rules to allow unauthenticated access to the /certificate endpoint match first, which breaks either the puppetserver or the client depending on the flavor of the request

Reply:
Ok, that makes sense, this is fine then